src/substrait/textplan/converter/Tool.cpp

```diff
@@ -1,6 +1,7 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 
 #include <getopt.h>
+#include <glob.h>
```

Comment:
Are either of these supported on Windows?

Reply:
Dropped getopt as it isn't needed yet and created a non-glob version for Windows (as compared to implementing an equivalent version with FindFirstFile).